sql/core/src/test/scala/org/apache/spark/sql/execution/debug/DebuggingSuite.scala

```diff
@@ -70,15 +70,15 @@ class DebuggingSuite extends SharedSparkSession with DisableAdaptiveExecutionSui
 
     val output = captured.toString()
     assert(output.replaceAll("\\[id=#\\d+\\]", "[id=#x]").contains(
-      """== BroadcastExchange HashedRelationBroadcastMode(List(input[0, bigint, false])), [id=#x] ==
-        |Tuples output: 0
-        | id LongType: {}
-        |== WholeStageCodegen (1) ==
-        |Tuples output: 10
-        | id LongType: {java.lang.Long}
-        |== Range (0, 10, step=1, splits=2) ==
-        |Tuples output: 0
-        | id LongType: {}""".stripMargin))
+"""== BroadcastExchange HashedRelationBroadcastMode(List(input[0, bigint, false]),false), [id=#x] ==
```

Comment:
can you keep the indentation?

Reply:
how to skip a "\n" in triple line string?

Reply:
if I keep origin indentation, it will exceed 100 column in lines.

Reply:
done